fvm/evm/stdlib/events.go

```diff
@@ -99,82 +82,60 @@ func (p *transactionEvent) ToCadence(location common.Location) (cadence.Event, e
 		errorMsg = p.Result.ValidationError.Error()
 	}
 
-	eventType := cadence.NewEventType(
-		location,
-		string(EventTypeTransactionExecuted),
-		transactionEventFields,
-		nil,
-	)
+	eventType := CadenceTypesForChain(chainID).TransactionExecuted
 
 	return cadence.NewEvent([]cadence.Value{
-		HashToCadenceArrayValue(p.Result.TxHash),
+		types.HashToCadenceArrayValue(p.Result.TxHash),
 		cadence.NewUInt16(p.Result.Index),
 		cadence.NewUInt8(p.Result.TxType),
-		BytesToCadenceUInt8ArrayValue(p.Payload),
+		types.BytesToCadenceUInt8ArrayValue(p.Payload),
 		cadence.NewUInt16(uint16(p.Result.ResultSummary().ErrorCode)),
 		cadence.String(errorMsg),
 		cadence.NewUInt64(p.Result.GasConsumed),
 		deployedAddress,
-		BytesToCadenceUInt8ArrayValue(encodedLogs),
+		types.BytesToCadenceUInt8ArrayValue(encodedLogs),
 		cadence.NewUInt64(p.BlockHeight),
-		HashToCadenceArrayValue(p.BlockHash),
```

Comment:
The refactor identified a mismatch in the event type declarations: The contract code (`contract.cdc) does not have the block hash parameter, but the `events.cdc` file does have it and the event emission includes the value for it here.

What is correct?

I remove the block hash value here, because the contract code (`contract.cdc`) does not have it.
If it should, I can revert the line here, and add the block hash back to `contract.cdc`. 
Please let me know!

Reply:
We removed the block hash recently, probably a left over.